lib/rucio/db/sqla/util.py

```diff
@@ -182,13 +187,17 @@ def create_root_account():
     identity4 = models.Identity(identity=ssh_id, identity_type=IdentityType.SSH, email=ssh_email)
     iaa4 = models.IdentityAccountAssociation(identity=identity4.identity, identity_type=identity4.identity_type, account=account.account, is_default=True)
 
+    # SAML authentication
+    identity5 = models.Identity(identity=saml_id, identity_type=IdentityType.SAML, email=saml_email)
```

Comment:
I think we have to remove these, and you have to add them to the unit-tests.
Currently the downgrade test is not running through, because we add the new SAML identities at setup and then obviously the downgrade run is failing. Can you please change that. Otherwise it seems good to me!